### PR TITLE
add FreeHeroes

### PR DESCRIPTION
**Pull request by heroesiiifan**

### CODE REVIEW

**Comment by mapron** [comment]
It's a bad Idea, a game is not even in an alpha stage, people will disappointed with it. Please, don't do it

File: games/f.yaml [hunk 1]
```diff
@@ -618,6 +618,27 @@
...
+  info: FreeHeroes is free engine for Heroes of Might and Magic 3.
+  langs:
+  - C++
+  licenses:
+  - MIT
+  content: commercial
+  originals:
+  - Heroes of Might and Magic III
+  repo: https://github.com/mapron/FreeHeroes
+  status: semi-playable
```
Comment (mapron) [inline games/f.yaml:638]: semi-playable is a direct lie.
You can emulate battles, true, but that 1/1000 part of 'playing' the game.

Till I am not finished adventure part I strongly against it.

**Comment by mapron** [comment]
ATM I working on the map generator (so it can be used with sod/hota). on prototype phase tho.
No way i get adventure mode in 2023, so advertising it as semi-playable now - feels like direct harm for my project. please stop.

**Comment by mapron** [comment]
As a compromise I could agree if you change 'semi-playable' to 'tech demo' or something.

File: games/f.yaml (hunk 1 above)
Comment (heroesiiifan) [reply games/f.yaml:638]: Well, there are three categories (as far as I know):

- playable
- semi-playable
- unplayable

Parts of the game are limited playable. That's why I thought semi-playable was the most appropriate. But I can change it to "unplayable" if you like.

**Comment by heroesiiifan** [comment]
> It's a bad Idea, a game is not even in an alpha stage, people will disappointed with it. Please, don't do it
Many games here have a much worse state of development than FreeHeroes.
Sorry. I thought it was good to point out here that the project exists.

> As a compromise I could agree if you change 'semi-playable' to 'tech demo' or something.
Should we change it to "unplayable"? Or do you not want an entry here? Then I will close the PR.

**Comment by mapron** [comment]
"unplayable" is ok for me too.

File: games/f.yaml [hunk 2]
```diff
@@ -618,6 +618,27 @@
...
+  info: FreeHeroes is free engine for Heroes of Might and Magic 3.
+  langs:
+  - C++
+  licenses:
+  - MIT
+  content: commercial
+  originals:
+  - Heroes of Might and Magic III
+  repo: https://github.com/mapron/FreeHeroes
+  status: unplayable
```
Comment (mapron) [inline games/f.yaml:638]: Thanks!

**Review by mapron — APPROVED** [review]

**Comment by heroesiiifan** [comment]
> Thanks for the compliment.  I appreciate it. Still don't want give false hopes (project was abandoned for 2 last years).

Thank you very much. Also for your efforts to keep Heroes III alive! :)

**Comment by mapron** [comment]
> Also for your efforts to keep Heroes III alive! :)
And thank you for spending your time on this MR.

File: games/f.yaml [hunk 3]
```diff
@@ -618,6 +618,27 @@
   updated: 2022-09-01
   info: FreeCS is a free-software reimplementation of Counter-Strike 1.5 running on the FTEQW engine.
 
+- name: FreeHeroes
+  development: active
```
Comment (cxong) [inline games/f.yaml:622]:
```suggestion
  development: sporadic
```

Comment (mapron) [reply games/f.yaml:622]: No objections from me, it's more accurate

**Review by cxong — APPROVED** [review]

**Pull request merged**